main/dhcp/dhclient-script.patch

```diff
@@ -1,30 +0,0 @@
---- a/client/scripts/linux
-+++ b/client/scripts/linux
-@@ -1,4 +1,4 @@
--#!/bin/bash
-+#!/bin/sh
```

Comment:
this patch needs to either be rebased or we need to add bash as a dependency for dhclient. I would prefer not depend on bash.

Reply:
That patch is gone. The `dhclient` package does depend on `bash` and rewriting the script so that it doesn't is a bit unnatural.

You wouldn't so the same thing with a `python` script, would you?